### ISSUE TITLE
Linearize API() not working in windows

### ISSUE THREAD

**Issue by arun3688** [OpenModelica/OpenModelica#8717]
### Description

Linearize() API is not working in windows when running with a mos script. We need explicitly use the `simulation executable` at runtime with the flag `-L` to generate the` linear_model.mo `   `(e.g)  ./vanderpol.exe -l 0`

### Version
`OpenModelica v1.19.0-dev-689-g37eb8d967f (64-bit)`

### Steps to Reproduce
```
loadString("
model VanDerPol  \"Van der Pol oscillator model\"
  Real x(start = 1, fixed = true);
  Real y(start = 1, fixed = true);
  parameter Real lambda = 0.3;
equation
  der(x) = y;
  der(y) = - x + lambda*(1 - x*x)*y;
end VanDerPol;"
); getErrorString();
linearize(VanDerPol); getErrorString() ;
```
The above mos script does not generate the `linear_model.mo` , however the simulation works

**Comment by mahge** [comment]
I think this has to do with the batch script I added. I am not sure what exactly is going wrong but it seems it is running simulation instead of linearization. I will see if I can fix it quickly.

**Comment by mahge** [comment]
Well that was quick. If nothing else goes wrong it should be fixed by #8721.

**Comment by arun3688** [comment]
> Well that was quick. If nothing else goes wrong it should be fixed by #8721.

thanks madher
